services/frontend/src/components/WeatherWidget.js

```diff
@@ -0,0 +1,73 @@
+import PropTypes from "prop-types";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faBolt,
+  faCloud,
+  faCloudSun,
+  faSun,
+  faWind,
+  faCloudShowersHeavy,
+  faCloudRain,
+  faSnowflake,
+  faSlash,
+} from "@fortawesome/free-solid-svg-icons";
+
+const WeatherWidget = ({ temp, cond }) => {
+  const getIconFromString = (conditionString) => {
```

Comment:
The weather icons are actually SVGs provided by the `FontAwesome` library. You can browse all the options here: https://fontawesome.com/v5.15/icons?d=gallery&p=2&q=weather&m=free

Note that the switch statement allows us to match a single icon for multiple different input strings.

Reply:
Why not a dictionary for this?
Presumably this could get even more complicated / messy, might be good to throw it into it's own file (or set of utility files; etc)

Reply:
Great point. Maybe a static JSON or yaml that's easier to configure and update without changing the source code.